### DISCUSSION TITLE
Prompts via API

### DISCUSSION THREAD

**Discussion by chris-spencer** [mudler/LocalAI#204] [Q&A]
Can we send a prompt via the API along with the question?

**Comment by ralyodio** [comment]
I'm also curious how I would build an api using localai in node.js

**Comment by mudler** [comment]
> Can we send a prompt via the API along with the question?

not sure I totally get that, you can send already prompts to the API. which endpoints are you talking about?

**Reply by chris-spencer** [reply]
Would you share a little example? Unless I've missed it, I couldn't see how to do that. (User error by the sounds of it)

I was hoping I could send a prompt template for example "prompt-template":"the following text is a copy of an article, I'd like you to summarise it {input}" then pass the input in another object "input":"text copied from the article would be here" 

Would you have an example of something similar? Or have I misunderstood? 

Many thanks
